nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -267,4 +269,10 @@ def get_compress_pipeline(
 class OVAWQAlgoAlgoBackend(OVWeightCompressionAlgoBackend):
```

Comment:
The usage of the `OVWeightCompressionAlgoBackend` seems not correct here. Is the `AWQAlgoBackend` correct?

Reply:
Done